frontend/public/components/monitoring/alerting.tsx

```diff
@@ -596,10 +593,15 @@ const alertStateToProps = (state: RootState, { match }): AlertsDetailsPageProps
 export const AlertsDetailsPage = withFallback(
   connect(alertStateToProps)((props: AlertsDetailsPageProps) => {
     const { alert, loaded, loadError, namespace, rule, silencesLoaded } = props;
-    const { annotations = {}, labels = {}, silencedBy = [] } = alert || {};
-    const { alertname, severity } = labels as any;
+    const annotations = alert?.annotations ?? {};
```

Comment:
You might want `||` instead of `??` in most of these cases. If you ever get `annotations: null`, could that cause an error?

Reply:
Thanks @spadgett. After looking at this again, I ended up refactoring in a way that removes the need for these 2 `??` anyway.